cli/upgrade.go

```diff
@@ -64,13 +64,14 @@ func defaultDownload(ctx context.Context, keyConjurerRcPath string) error {
 		return err
 	}
 
-	binary, err := client.GetLatestBinary(ctx)
+	f, err := os.OpenFile(keyConjurerRcPath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0744)
 	if err != nil {
-		return fmt.Errorf("unable to download the latest binary: %w", err)
+		return fmt.Errorf("unable to open %q: %w", keyConjurerRcPath, err)
 	}
 
```

Comment:
Nit: This is a suboptimal pattern, replacing the binary in place. We should probably handle this in a safer way in the future.